src/main/java/seedu/address/logic/commands/ExportEventXmlCommand.java

```diff
@@ -0,0 +1,139 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.File;
+import java.util.List;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.w3c.dom.Attr;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Export an Event information as an XML
+ */
+public class ExportEventXmlCommand extends Command {
+
+    public static final String COMMAND_WORD = "exporteventxml";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": exports an XML file of the event "
+            + "given the file path to the xml file\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    private static final String MESSAGE_EXPORT_EVENT_SUCCESS = "Event at %1$d exported"
+            + "to your Desktop.";
+    private static final String MESSAGE_EXPORT_EVENT_FAILED = "Event export failed, please try again.";
+
+    private final Index index;
+    private final String exportTypeE = "EVENT";
+
+    /**
+     * @param index of volunteer or event in the filtered list
+     */
+    public ExportEventXmlCommand(Index index) {
+        requireNonNull(index);
+        this.index = index;
+    }
+
+    @Override
+    public CommandResult execute(Model model, CommandHistory history) throws CommandException {
+        requireNonNull(model);
+        List<Event> list = model.getFilteredEventList();
+
+        // Handle case where the index input exceeds or equals the size of the last displayed list
+        if (index.getZeroBased() >= list.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
+        }
+
+        // Get the Event object whom the index corresponds to
+        Event selectedEvent = list.get(index.getZeroBased());
+
+        try {
+            createEventXml(model, selectedEvent);
+        } catch (Exception e) {
```

Comment:
It might be useful to find out what type(s) of Exception may result from running this method call, and handling them more specifically

Reply:
Used it as a blanket statement for now because of possible file creation exceptions when handling the xml transformer and writing throws out to an error at creating the file. I will update the individual exceptions accordingly although they have the same handler